orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -89,14 +97,24 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
       String notificationState
       ExecutionStatus executionStatus
 
+      if (manualJudgmentNavigation) {
+        checkForAnyParentExecutions(stage)
```

Comment:
The naming of this method implies no side effects, I'd suggest a change of name.